test/tests/api/v2_0/catalogs_tests.py

```diff
@@ -0,0 +1,46 @@
+from config.api2_0_config import *
+from modules.logger import Log
+from on_http_api2_0 import ApiApi as Api
+from proboscis.asserts import assert_equal
+from proboscis.asserts import assert_raises
+from proboscis.asserts import assert_not_equal
+from proboscis.asserts import assert_is_not_none
+from proboscis import SkipTest
+from proboscis import test
+from json import dumps, loads
+from on_http_api2_0 import rest
+
+
+LOG = Log(__name__)
+
+
+@test(groups=['catalogs_api2.tests'])
+class CatalogsTests(object):
+    def __init__(self):
+        self.__client = config.api_client
+
+    @test(groups=['catalogs_api2.tests', 'api2_check-catalogs'])
+    def test_catalogs(self):
+        """Testing GET:api/2.0/catalogs to get list of catalogs"""
+        Api().get_catalog()
+        rsp = self.__client.last_response
+        assert_not_equal(404, rsp.status, message=rsp.reason)
```

Comment:
@srinia6 This check seems unnecessary.  I think the `assert_equal(200, rsp.status, ...)` check is sufficient by itself.